fe1-web/src/features/home/navigation/HomeNavigation.tsx

```diff
@@ -40,67 +31,36 @@ const HomeNavigation = () => {
         title: STRINGS.home_navigation_title,
         Component: Home,
         tabBarIcon: makeIcon('list'),
-        order: -99999999,
+        headerLeft: () => null,
+        headerRight: HomeHeaderRight,
       } as HomeFeature.HomeScreen,
       {
-        id: STRINGS.navigation_home_mock_connect,
-        title: STRINGS.navigation_app_connect,
+        id: STRINGS.navigation_home_connect,
+        title: STRINGS.navigation_home_connect,
         Component: ConnectNavigation,
-        tabPress: (e) => {
-          // prevent navigation
-          e.preventDefault();
-          navigation.navigate(STRINGS.navigation_app_connect);
-        },
-        tabBarIcon: makeIcon('scan'),
-        order: -10000,
-      } as HomeFeature.HomeScreen,
+        headerShown: false,
+      },
       // sort screens by order before rendering them
-    ].sort((a, b) => a.order - b.order);
```

Comment:
How is it possible that we can remove orders of screens? Is it handled automatically?

Reply:
Since it now is a stack navigator (i.e. no bottom tab navigation bar), an order would not matter :)